docs/log-correlation.asciidoc

```diff
@@ -94,3 +94,27 @@ end
 # [2019-09-16 11:54:59 +0200][RailsTestApp][INFO][transaction.id=3b92edcccc0a6d1e span.id=f3d7e32f176d4c93 trace.id=1275686e35de91f776557637e799651e]   Rendered text template (Duration: 0.1ms | Allocations: 17)
 # [2019-09-16 11:54:59 +0200][RailsTestApp][INFO][transaction.id=3b92edcccc0a6d1e span.id=3bde4e9c85ab359c trace.id=1275686e35de91f776557637e799651e] Completed 200 OK in 1ms (Views: 0.3ms | Allocations: 187)
 ----
+
+==== Extracting trace IDs from the logs
```

Comment:
ci failed on this heading:
```
06:37:52 INFO:build_docs:asciidoctor: WARNING: log-correlation.asciidoc: line 98: section title out of sequence: expected level 2, got level 3
```
This needs to either be a level 2 heading:
```
=== Extracting trace IDs from the logs
```

Or, and this is preferable in this case, it needs to be a level three heading that is floated to this page:
```suggestion
[float]
==== Extracting trace IDs from the logs
```

A `[float]` tag allows you to use multiple, same level headings on the same page. Without the `[float]` tag, Asciidoctor thinks you want the content on a new page. In that case, the heading level needs to one less to indicate a new page.